app/index.js

```diff
@@ -83,31 +83,31 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'name',
         message: 'What\'s your name:',
         default: this.options.name || this.gitc.user.name,
-        when: this.options.name == null
+        when: !this.options.name
```

Comment:
the `== null` checks fail the eslinter and it's bad practice in general, as it's ambiguous. I will adjust so it won't prompt with empty string.